mdformat_tables/plugin.py

```diff
@@ -80,8 +79,10 @@ def render_token(
     # gather all cell tokens into row * column array
     rows = []
     align = []
-    while index < len(tokens) and tokens[index].type != "table_close":
```

Comment:
The `index < len(tokens)` part contains an off-by-one error here.

If `index == len(tokens) - 1` (the upper bound), then on the following line (84), we increment index by one, and the line after that (85) will IndexError.